controller/volume_controller.go

```diff
@@ -3198,6 +3198,11 @@ func (c *VolumeController) createEngine(v *longhorn.Volume, isNewEngine bool) (*
 
 	if isNewEngine {
 		engine.Spec.Active = true
+	} else {
+		engine.Status.Snapshots = *snapshots
```

Comment:
1. Will a CR `Create` API include the status info?
2. I prefer to somehow skip the snapshot syncing before an engine becomes the main/active one.

Reply:
Yes, you are right, it is weird to have the status info when creating a CR.
Removed it, and it will still wait util the engine is running and engine monitor starts to synchronize snapshots.